internal/store/store.go

```diff
@@ -0,0 +1,47 @@
+package store
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type Store struct {
+	*pgxpool.Pool
+}
+
+func NewStore(ctx context.Context, connString string) (Store, error) {
```

Comment:
Мне кажется тут стоило  бы использовать тип указателя и для возращаемого значения и для ресиверов. Во-первых так будет проще проверять невалидное значение на стороне вызывающего кода. Во-вторых если частью нашей структуры являются не примитивные типы (в нашел случае `*pgxpool.Pool`) лучше избегать копирования если нет явной необходимости. 

```suggestion
func NewStore(ctx context.Context, connString string) (*Store, error) {
```